### DISCUSSION TITLE
Can't display OGP image

### DISCUSSION THREAD

**Discussion by ryuapp** [freshframework/fresh#1760] [Q&A]
Hello.
Fresh site can't display OGP image.
https://fresh.deno.dev/
But for some reason the docs displays the image.
https://fresh.deno.dev/docs/introduction

Is this bug in Fresh or Deno Deploy?

**Comment by marvinhagemeister** [comment]
What's an OGP image?

**Reply by ryuapp** [reply]
Sorry, I don't know the appropriate words in English.
"OGP image" I said means `og:image` in meta tag.
It refers to the thumbnail image that is displayed when a URL is detected on SNS.
ex. `<meta property="og:image" content="https://example.com/og/image.png">`

On Fresh site, both have the same <meta  property="og:image"> tag, but only the docs has the image displayed.
Check following this screenshot or paste `https://fresh.deno.dev/` in Discord.
![image](https://github.com/denoland/fresh/assets/114303361/ded98dcf-fcaa-4fb1-9ec0-60d958ba180b)

**Comment by ryuapp** [comment]
The OG image in `https://fresh.deno.dev/` is not displayed because it will redirect all other than the specific user agent when it doesn't have any header.
